docs/GDI-SOP_ISM.md

```diff
@@ -0,0 +1,176 @@
+5. OC/SDPC members from the node accessions ([7.2](#72-gdi-sop-accessioning-and-file-naming)) the approved SOP according to agreed process and the SOP goes into production
+
+![Valid: Proposal for creating and approving node-specific SOP templates ](images/GDI-ISM_schema_Node_Instance.png)
+
+_Figure 3: Proposal for creating and approving node-specific SOPs instances._
+
+SOP template reviews and revisions should follow guidelines for GDI SOP revision and review ([7.3](#73-gdi-sop-review-and-revision)) in the GDI SOP GitHub repository.
+
+### 7.2 GDI SOP accessioning and file naming
+
+**GDI SOP templates** will be assigned accessions upon being approved. GDI SOPs template accessions will take the form “GDI-SOP00000.v0”, where the number before the full stop will auto increment by 1 and the number after the full stop will indicate the version number. For example, version 1 of the first approved GDI SOP template will be given the accession “GDI-SOP00001.v1”. GDI SOP templates must be referenced by this accession.
+
+**Node instances** of GDI SOP templates will be assigned accessions which take the form of the template accession followed by a string (“node identifier”) chosen by each GDI node to represent their node’s SOPs. For example, if the Swedish node chooses “SWE” as their node identifier, then their instance of version 1 of the first approved GDI SOP template will be given the accession “GDI-SOP00001.v1-SWE”. Nodes can modify their SOP instances - as long as SOP instances adhere to the requirements of the originating template - and append additional versioning to the accession. For example, if the Swedish node decides to make a change to their GDI-SOP00001.v1-SWE SOP (assuming that they continue to adhere to the GDI-SOP00001.v1 template), they may choose to name it “GDI-SOP00001.v1-SWE.v1”.
+
+Filenames for GDI SOP templates and node instances must contain the accession as the first part of the filename. The remainder of the filename can vary but must be separated from the accession with an underscore (“\_”). For example, if the GDI SOP “GDI-SOP00001.v1” is about how to manage node helpdesk tickets, the file for the template could be named “GDI-SOP00001.v1_Node-Helpdesk-Ticket-Management-Template”.
+
+GDI-SOP00001.v1_Node-Helpdesk-Ticket-Management-Template-SWE.v1
+
+**Supporting GDI SOP documents** will also be assigned an accession number…
+
+Filenames for GDI SOP supporting documents must start with “GDI-SOP\_”, followed by a brief text describing the document, a full stop, and finally a version number. For example, the first approved version of the GDI SOP Charter could be named “GDI-SOP_Charter.v1”.
+
+### 7.3 GDI SOP review and revision
+
+<_To do- describe a general plan for how GDI SOP templates, node instances of SOPs, and supporting documentation will be reviewed and revised. Include guidance for how to update “Document History” sections of all these documents._>

```

Comment:
I would have liked for this to be elaborated slightly, we could at least mention the checklist I created here: https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_review-checklist.md